src/h/rexterns.h

```diff
@@ -419,3 +419,9 @@ extern struct  b_cons *LstTmpFiles;
 #ifdef Audio
 extern int isPlaying;
 #endif					/* Audio */
+
+#ifdef VerifyHeap
+extern long vrfy;
+extern void vrfyLog(const char *fmt, ...);
+extern void vrfy_Live_Table(struct b_table *b);
```

Comment:
Where are these functions? I don't see their implementation as part of this PR?

Reply:
Found it! Github wasn't showing rmemmgt.r by default because  the diff is large!